contracts/capabilities/Blacklistable.sol

```diff
@@ -0,0 +1,89 @@
+pragma solidity 0.5.16;
+
+import "../roles/BlacklisterRole.sol";
+
+/**
+Keeps track of Blacklists and can check if sender and reciever are configured to allow a transfer.
+Only administrators can update the Blacklists.
+ */
+contract Blacklistable is BlacklisterRole {
+    // Track whether Blacklisting is enabled
+    bool public isBlacklistEnabled;
+
+    // The mapping to keep track if an address is blacklisted
+    mapping (address => bool) public addressBlacklists;
+
+    // Events to allow tracking add/remove.
+    event AddressAddedToBlacklist(address indexed addedAddress, address indexed addedBy);
+    event AddressRemovedFromBlacklist(address indexed removedAddress, address indexed removedBy);    
+    event BlacklistEnabledUpdated(address indexed updatedBy, bool indexed enabled);
+
+    function _setBlacklistEnabled(bool enabled) internal {
+        isBlacklistEnabled = enabled;
+        emit BlacklistEnabledUpdated(msg.sender, enabled);
+    }
+
+    /**
+    Sets an address's blacklisting status.  Only administrators should be allowed to update this.
+     */
+    function _addToBlacklist(address addressToAdd) internal {
+        // Verify a valid address was passed in
+        require(addressToAdd != address(0), "Cannot add address 0x0 to the blacklist.");
+
+        // Set the address's white list ID
+        addressBlacklists[addressToAdd] = true;
+
+        // Emit the event for new Blacklist
+        emit AddressAddedToBlacklist(addressToAdd, msg.sender);
+    }
+
+    /**
+    Clears out an address from the blacklist.  Only administrators should be allowed to update this.
+     */
+    function _removeFromBlacklist(address addressToRemove) internal {
+        // Verify a valid address was passed in
+        require(addressToRemove != address(0), "Cannot remove address 0x0 from the blacklist.");
+        
```

Comment:
added a check that the address is blacklisted?

Reply:
Added a check.